tests/unit/source_code/test_files.py

```diff
@@ -68,7 +68,7 @@ def test_files_walks_directory():
     languages.linter(Language.PYTHON).lint.return_value = [Mock(code='some-code')]
     languages.fixer.return_value = None
     files = Files(languages)
-    path = Path(__file__).parent
+    path = Path(Path(__file__).parent.parent, "aws/")
```

Comment:
seems unrelated but fails with this PR because source_code folder now contains non-python files

Reply:
doesn't seem to be relevant.

Reply:
see my comment, this PR breaks the test